packages/core/src/components/fl-badge/readme.md

```diff
@@ -0,0 +1,22 @@
+# fl-badge
+
+
+
+<!-- Auto Generated Below -->
+
+
+## Properties
+
+| Property     | Attribute    | Description | Type                                                                                                         | Default          |
```

Comment:
add descriptions

Reply:
Didn't see this file in my pull, check updated spinner and badge. I'm happy to add that with the descriptions if that's what you want.